README.md

```diff
@@ -102,6 +102,8 @@ web-server                1/1       Running   0          1m
 
 ### Snapshot Example
 
+Please note that VolumeSnapshot is currently alpha feature. In order to enable this feature, please install CSI driver with
+dev version by setting GCE_PD_DRIVER_VERSION=dev
```

Comment:
can you clarify that this is an environment variable and also surround the variable itself with "`"

Reply:
fixed. PTAL